app/src/main/java/com/example/inventory/ui/home/HomeScreen.kt

```diff
@@ -61,137 +61,141 @@ object HomeDestination : NavigationDestination {
 /**
  * Entry route for Home screen
  */
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
 @Composable
 fun HomeScreen(
     navigateToItemEntry: () -> Unit,
     navigateToItemUpdate: (Int) -> Unit,
-    modifier: Modifier = Modifier,
-    viewModel: HomeViewModel = viewModel(factory = AppViewModelProvider.Factory)
+    modifier: Modifier = Modifier
 ) {
-    val homeUiState by viewModel.homeUiState.collectAsState()
+    val scrollBehavior = TopAppBarDefaults.enterAlwaysScrollBehavior()
+
     Scaffold(
+        modifier = modifier.nestedScroll(scrollBehavior.nestedScrollConnection),
         topBar = {
             InventoryTopAppBar(
                 title = stringResource(HomeDestination.titleRes),
-                canNavigateBack = false
+                canNavigateBack = false,
+                scrollBehavior = scrollBehavior
             )
         },
         floatingActionButton = {
             FloatingActionButton(
                 onClick = navigateToItemEntry,
-                modifier = Modifier.navigationBarsPadding()
+                shape = MaterialTheme.shapes.medium,
```

Comment:
does this not default to medium shape? it should per https://m3.material.io/styles/shape/shape-scale-tokens

Reply:
Yes this may be a bug. Unless I set the shape explicitly it is not setting to the custom shape.

Reply:
yeah makes sense - I'll investigate more but this is ok for now